### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by omsoo**
Updated the icons to fit the design
Used font-awesome icons
Updated the google font
Added background image

#Impact
All features added were tested with no error noticed

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -115,455 +115,3 @@ a {
   color: white;
   margin-top: 1.2rem;
 }
```
Comment (keroloussamy) [inline style.css:117]: - [ ] Kindly add style to remove the dots from the list, to be like the design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -58,209 +58,5 @@ <h3 class="hero-highlight-2">I’am a Software Developer</h3>
         />
```
Comment (keroloussamy) [inline index.html:58]: - [ ] Kindly add the background image using the `URL()` role in CSS, because here the image is not an element, it's more semantic.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @omsoo, 

Good job so far, And I want to approve it ✌️ but 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi @omsoo, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

## OPTIONAL - I can see a horizontal scrolling of your mobile version - I fix it with `width: 100%` for `.hero-container` and `.navbar` 👍 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**